src/app/modules/project/embed-button/embed-button.component.ts

```diff
@@ -0,0 +1,49 @@
+import { EmbedService } from './../../../services/embed.service';
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { HttpErrorResponse } from '@angular/common/http';
+import { EmbeddedProject } from 'src/app/models/domain/embedded-project';
+import { environment } from 'src/environments/environment';
+
+
+@Component({
+  selector: 'app-embed-button',
+  templateUrl: './embed-button.component.html',
+  styleUrls: ['./embed-button.component.scss']
+})
+/**
+ * Button that generates an embedded project and creates an iframe example link
+ */
+export class EmbedButtonComponent implements OnInit {
+  EmbeddedProject: EmbeddedProject;
```

Comment:
We should be explicit with access modifiers, public and private should be annotated everywhere.